uint8 iskeepCurSpeed = 1
uint8 notkeepCurSpeed = 0

float64 EgoSpeed
float64 PreSpeed
float64 LimitSpeed
float64 Dis2Stopline

uint8 intersectionID

float64 Latitude
float64 Longitude
float64 Height


float64 upperSpeed
float64 bottomSpeed
uint8 keepCurSpeed

float64 StrColor
float64 StrlikelyEndTime

float64 LeftColor
float64 LeftlikelyEndTime
